src/engine/source/kvdb/src/kvdbHandler.cpp

```diff
@@ -144,20 +144,51 @@ std::variant<std::string, base::Error> KVDBHandler::get(const std::string& key)
     return base::Error {"Cannot access RocksDB::DB"};
 }
 
-std::variant<std::unordered_map<std::string, std::string>, base::Error> KVDBHandler::dump()
+std::variant<std::list<std::pair<std::string, std::string>>, base::Error> KVDBHandler::dump(const uint32_t page,
```

Comment:
What is the logic behind calling with page and records equal to 0?. This should be documented.

Reply:
This is going to be used so that internally we avoid pagination, we are going to document this in the function.